.github/workflows/check-links.yml

```diff
@@ -7,15 +8,22 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
+
+concurrency:
+  group: links-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
Doesn't allow for the same workflow/action to run multiple times in the same PR.

Doesn't apply to `R CMD check` workflow nor `vbump`